README.md

```diff
@@ -29,17 +29,42 @@ video/x-raw
-| **stop-error**        | Boolean             |    true | Stop the gstreamer if kernel modules fail to open.                   |
-| **max-video-rate**    | Float [0.001 - 120] |     120 | Force maximum video frame rate using thread sleeps.                  |
-| **max-drpai-rate**    | Float [0 - 120]     |     120 | Force maximum DRPAI frame rate using thread sleeps.                  |
-| **smooth-video-rate** | Float [1 - 1000]    |       1 | Number of last video frame rates to average for a more smooth value. |
-| **smooth-drpai-rate** | Float [1 - 1000]    |       1 | Number of last DRPAI frame rates to average for a more smooth value. |
+### General Parameters
+
+| Name                     | Type                 | Default | Description                                                                                               |
+|--------------------------|----------------------|--------:|-----------------------------------------------------------------------------------------------------------|
+| **model**                | String               |     --- | (Required) The name of the pre-trained model and the directory prefix.                                    |
+| **multithread**          | Boolean              |    true | Use a separate thread for object detection.                                                               |
+| **log-detects**          | Boolean              |   false | Print detected objects in standard output.                                                                |
+| **log-server**           | Host:Port (String)   |     --- | Address of logs to send in UDP messages in [JSON format](JSON.md) to the specified port on a remote host. |
+| **show-fps**             | Boolean              |   false | Render frame rates of video and DRPAI at the corner of the video.                                         |
+| **stop-error**           | Boolean              |    true | Stop the gstreamer if kernel modules fail to open.                                                        |
+| **max-video-rate**       | Float [0.001 - 120]  |     120 | Force maximum video frame rate using thread sleeps.                                                       |
+| **max-drpai-rate**       | Float [0 - 120]      |     120 | Force maximum DRPAI frame rate using thread sleeps.                                                       |
+| **smooth-video-rate**    | Float [1 - 1000]     |       1 | Number of last video frame rates to average for a more smooth value.                                      |
+| **smooth-drpai-rate**    | Float [1 - 1000]     |       1 | Number of last DRPAI frame rates to average for a more smooth value.                                      |
+| **smooth-bbox-rate**     | Float [1 - 1000]     |       1 | Number of last bounding-box updates to average. (requires tracking)                                       |
```

Comment:
Why is the default set to 1? Does this make sense? 🤔  What values have you found provide a good experience, maybe that should be the default instead, right?

Reply:
> Why is the default set to 1? Does this make sense? 🤔 What values have you found provide a good experience, maybe that should be the default instead, right?

This applies to all default values.

Reply:
The value 1 means no smoothing because it only uses the last 1 to determine the result.
Think about it. When someone says let's average out only 1 frame, it would be as if we are not doing anything.

Right?

Reply:
It depends on how you define the "good experience". If objects are moving slowly, the smoother the better. If objects are moving fast, smoothness can cause a delay in the bounding box movement. Read more about the moving average concept here: https://en.wikipedia.org/wiki/Moving_average.

So I think it shouldn't be used unless the user asks for it.